base/record_replay.cc

```diff
@@ -111,6 +117,30 @@ void NewCheckpoint() {
   OP(V8RecordReplayNewCheckpoint());
 }
 
+size_t NewBookmark() {
```

Comment:
One comment: I changed my mind on this.  I think it's better for this to be `uint64_t` as you had it before.  `size_t` is a platform specific type and  a bookmark is explicitly a 64-bit unsigned integer.  So let's be explicit in the types.

Sorry about the misdirection.